packages/plugin-react-swc/src/index.ts

```diff
@@ -151,43 +147,13 @@ const react = (_options?: Options): PluginOption[] => {
         if (!result) return
         if (!refresh) return result
 
-        const hasRefresh = refreshContentRE.test(result.code)
-        if (!hasRefresh && !reactCompRE.test(result.code)) return result
-
         const sourceMap: SourceMapPayload = JSON.parse(result.map!)
```

Comment:
There is a slight behaviour change here: sourceMap is parsed for non 'refreshed' files

Reply:
Ah, I thought about that and moved the `JSON.parse` into `addRefreshWrapper` and then forgot about removing it here.